source/CommonJobs/CommonJobs.Mvc.UI/Views/Shared/_Layout.cshtml

```diff
@@ -39,53 +38,26 @@
     <link rel="shortcut icon" href="@Url.Content("~/Content/Images/favicon.ico")" />
 </head>
 <body>
-    <div class="bootstrap-scope navbar-container">
-        <div class="navbar navbar-inverse navbar-fixed-top">
-            <div class="navbar-inner">
-                <a class="brand" href="@Url.Content("~")">CommonJobs</a>
-                <button type="button" class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <ul class="nav nav-right">
-                    @if (Request.IsAuthenticated && User != null && User.Identity != null && User.Identity.Name != null) 
-				    {
-                        <li class="username"><small>@User.Identity.Name
+    <header class="header__main">
+        <div class="header__holder">
+            <div class="suit-wrapper">
+                <a href="@Url.Content("~")" class="logo">
+                    <img src="/Content/Images/Evaluations/header_logo.png" alt="CommonJobs">
+                </a>
+                <nav class="header__sub">
+                    <span class="header__user">
+                        @User.Identity.Name
                         @if (User.Identity.AuthenticationType == "Forms")
                         {
-                            <text>[</text>
-                            @Html.ActionLink("LogOff", "LogOff", "Account")
-                            <text>]</text>
+                            <text>[<a href="@Url.Content("~/Account/LogOff")">LogOff</a>]</text>
                         }
-                        </small></li>
-				    }
-                    else
-                    {
-                        //TODO: show link to logon page when AuthenticationType == "Forms"
-                    }
-                    @Html.Partial("EnvironmentBanner")
-                </ul>
-                
-                <div class="nav-collapse collapse">
-                <ul class="nav">
-                    @Html.NavItem("Postulantes", "Index", "Applicants")
```

Comment:
Would you mind to send me a screenshot of how the system looks with the new menu?

I imagine that you guys made some other way of navigating through the sections, and I'd just like to see it. If you didn't, that should be part of this PR so that the system does not become unusable.